src/jaxsim/api/data.py

```diff
@@ -738,7 +740,14 @@ def random_model_data(
     ω_max = jnp.array(base_vel_ang_bounds[1], dtype=float)
     ṡ_min, ṡ_max = joint_vel_bounds
 
-    random_data = JaxSimModelData.zero(model=model)
+    random_data = JaxSimModelData.zero(
+        model=model,
+        **(
+            dict(velocity_representation=velocity_representation)
+            if velocity_representation is not None
+            else {}
+        ),
```

Comment:
Is there any particular reason why you are unpacking a single item dictionary?
```suggestion
        velocity_representation=(
            velocity_representation if velocity_representation is not None else {}
        ),
```

Reply:
Alright, thanks